### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Nikita211112**

### CODE REVIEW

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sum = 0;
```
Comment (polosanya) [inline src/calculateRentalCost.js:31]: Create a const variable for `days * 40`. You duplicate this value a lot
image

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -28,7 +28,17 @@
...
+
+  if (days >= 7) {
+    sum = (days * 40) - 50;
+  } else if (days >= 3 && days < 7) {
+    sum = (days * 40) - 20;
+  } else {
+    sum = days * 40;
```
Comment (polosanya) [inline src/calculateRentalCost.js:38]: No need to use if...else. It makes your code a bit complicated. 
Just return the full price immediately if the condition is true
image

**Review by polosanya — CHANGES_REQUESTED** [review]
Almost done! Let's improve your code a bit)
Create variables for all main values (40, 50, 20 etc.)

**Review by loralevitska — APPROVED** [review]
🔥

**Pull request closed without merging**